packages/workspace/src/generators/ci-workflow/ci-workflow.spec.ts

```diff
@@ -337,18 +337,23 @@ describe('CI Workflow generator', () => {
                   filter: tree:0
                   fetch-depth: 0
 
-              # This enables task distribution via Nx Cloud
```

Comment:
This should not have moved.

Reply:
It hasnt been removed the line number just changed